storybookjs/storybook
# Addon-knobs: Fix search params with URI encoding

Issue: #11641

## What I did

This change applies basic URI encoding to knob keys/values in `<iframe>` URL and torn-away window URL in preview.

## How to test

Use the newly added Knobs - withKnobs - With Knob Value To Be Encoded story in `examples/official-storybook`, put `10% 20% 30%` to the knob value, click on torn-away icon (icon with square and up arrow), and see if the content is right.
